### PR TITLE
Android Support Repository -> Local Maven repository for Support Libr…

### PR DESCRIPTION
**Pull request by sunnylqm**

### CODE REVIEW

**Comment by facebook-github-bot-5** [comment]
By analyzing the blame information on this pull request, we identified @satya164 and @mkonicek to be potential reviewers.

File: docs/AndroidBuildingFromSource.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Make sure you have the following installed:
 
 1. Android SDK version 23 (compileSdkVersion in [`build.gradle`](https://github.com/facebook/react-native/blob/master/ReactAndroid/build.gradle))
 2. SDK build tools version 23.0.1 (buildToolsVersion in [`build.gradle`](https://github.com/facebook/react-native/blob/master/ReactAndroid/build.gradle))
-3. Android Support Repository >= 17 (for Android Support Library)
+3. Local Maven repository for Support Libraries (former name `Android Support Repository`) >= 17 (for Android Support Library)
```
Comment (satya164) [inline docs/AndroidBuildingFromSource.md]: `former name` -> `formerly`

Comment (sunnylqm) [inline docs/AndroidBuildingFromSource.md]: Or 'formerly named' ? I am not a native speacker...

Comment (satya164) [inline docs/AndroidBuildingFromSource.md]: Both work :)

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-1** [comment]
@sunnylqm updated the pull request.

**Comment by facebook-github-bot-3** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1537560106548920/int_phab) to review.

**Pull request closed without merging**